ivy/functional/frontends/tensorflow/raw_ops.py

```diff
@@ -64,6 +64,25 @@ def ApproximateEqual(
     return ret < tolerance
 
 
+@to_ivy_arrays_and_back
+def AvgPool(
+    *,
+    value,
+    ksize,
+    strides,
+    padding,
+    data_format="NHWC",
+    name="AvgPool",
+):
+    return ivy.avg_pool1d(
```

Comment:
Judging from `AvgPool`'s documentation it seems that it actually does 2d average pooling, so you'd need to call `ivy.avg_pool2d`.